paasta_tools/kubernetes_tools.py

```diff
@@ -1591,11 +1593,26 @@ def get_crypto_secret_name(self) -> str:
             self.get_namespace(), "crypto-key", self.get_service(), self.get_instance()
         )
 
-    def get_boto_secret_signature_name(self) -> str:
+    def get_boto_new_secret_signature_name(self) -> str:
+        """
+        Keeping in case we will want to canonize secret naming
+        """
```

Comment:
probably easier maintenance-wise to bring this back whenever do we want to go about doing this rather than leaving unused code in `master` :)